Sources/NostrSDK/NIP44v2Encrypting.swift

```diff
@@ -0,0 +1,335 @@
+//
+//  NIP44v2Encrypting.swift
+//
+//
+//  Created by Terry Yiu on 3/16/24.
+//
+
+import Foundation
+import Clibsodium
```

Comment:
`import Clibsodium` breaks the documentation generation step of the build and I'm not sure why.

```
error: 'NostrSDK' does not contain any documentable symbols or a DocC catalog and will not produce documentation
```